src/_pytest/terminal.py

```diff
@@ -231,7 +231,7 @@ def pytest_configure(config: Config) -> None:
     config.pluginmanager.register(reporter, "terminalreporter")
     if config.option.debug or config.option.traceconfig:
 
-        def mywriter(tags, args):
```

Comment:
The `tags` arg is probably required because that's how `setprocessor` calls it.

Reply:
ah good spot which is pluggy-land I think, will revert :+1: